queries/orderbook/barn_batch_rewards.sql

```diff
@@ -48,6 +48,10 @@ order_surplus AS (
             WHEN o.kind = 'sell' THEN t.buy_amount - t.sell_amount * o.buy_amount / (o.sell_amount + o.fee_amount)
             WHEN o.kind = 'buy' THEN t.buy_amount * (o.sell_amount + o.fee_amount) / o.buy_amount - t.sell_amount
         END AS surplus,
+        CASE
+            WHEN o.kind = 'sell' THEN t.buy_amount - t.sell_amount * oq.buy_amount * (1 - 1 / oq.sell_amount * oq.gas_amount * oq.gas_price / oq.sell_token_price) / oq.sell_amount
```

Comment:
This is hard to parse. I would rearrange the terms a bit so that it makes a bit more sense. Also, i am always confused with the implicit order when omitting parenthesis so i would make this explicit as well. Maybe something like

`THEN t.buy_amount - (t.sell_amount / oq.sell_amount) * oq.buy_amount * (oq.sell_amount - oq.gas_amount * oq.gas_price / oq.sell_token_price)` ?